### ISSUE TITLE
Semantics of URI for concept resources

### ISSUE THREAD

**Issue by damooo** [atomicdata-dev/atomic-data-docs#131]
Current spec as it exists, doesn't distinguish between information resources and concept resources.

I.e, as every subject's identifier has to deref to an information resource representation(like json-ad doc), the architecture is
- either effectively limited only to self describing information resources / documents. 
- Or, has issue in that, it cannot distinguish a resource (like person), and document describing him (available at same url) 

From docs, it seems second case. This may be a serious ontological issue for serious usage. And also makes the data semantically not a subset of rdf as claimed.

**Comment by damooo** [comment]
It may be resolved by saying all concept resource ids MUST end with `#`, while information resource ids do not. And id of information resource describing a concept resource MUST be same fragment-less part.

This way, there is distinction between a concept resource, and information resource describing it, all the while allowing deererencing affordance of atomic data architecture, and having no semantic clashes with rdf.

**Comment by joepio** [comment]
> (Atomic Data is) limited only to self describing information resources / documents.

I agree that it should be possible to share information about a thing that is hosted somewhere else. However, _re-using_ that information becomes a challenge, as the ones who know only of the original resource will not find the newly added information.

Anyways, I think this problem could probably be solved by defining a Property that does what you want it to do. Maybe it could be called `describes`, and it's definition could be: `The URI of the thing that is described by this resource. This can be a link that does not resolve to any Atomic Data Document.`

**Comment by damooo** [comment]
Currently all subject urls are mandated to be dereferable. That is the issue for concept resources.

**Comment by joepio** [comment]
I think I'm not completely understanding the problem then. What are `concept resources`? I wasn't able to find this on the internet, let alone the RDF spec. 

Could you describe a usecase example?

**Comment by damooo** [comment]
Following documents can give proper context. And in general helpful for understanding uri.

- [Four Uses of a URL: Name, Concept, Web Location and Document Instance](https://www.w3.org/2002/11/dbooth-names/dbooth-names_clean.htm)
- [Cool Uris for semweb](https://www.w3.org/TR/cooluris/)
- [What do HTTP URIs Identify?](https://www.w3.org/DesignIssues/HTTP-URI.html)

**Comment by joepio** [comment]
Thanks for sharing the links, I think I'm familiar with most of them, but I still don't understand what the issue is. I think you're referring to Tim Beners-Lee's `ConceptualWork`, right? If you want to make a distinction between resources that represent something and resources that _describe_ something, then I think my suggested `describes` property would cover the usecases I have in mind.

Can you provide a usecase for a `concept resource` and describe your wanted outcome? The more concrete the example the better!
